.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "panda"]
 	path = panda
-	url = ../../commaai/panda.git
```

Comment:
Can't merge this obviously, are the needed PRs open on panda and opendbc?

Reply:
yes they are. but also see above for more complex needs